### ISSUE TITLE
Rollertest aarch64/arm linux compile

### ISSUE THREAD

**Issue by PublicVoidUpdate** [rollerozxa-archive/rollertest#3]
Compiling for arm linux results in an irrlicht error, (using your library), i understand that this project is intended for android, but is it possible to compile it for linux similar to the official minetest? i have a pinephone i want to get minetest on, but of course minetest isn't very touch friendly.

**Comment by rollerozxa** [comment]
Yes, it should be possible to build this for any platform Minetest supports, not just Android. Are you checking out the correct IrrlichtMt tag? It should be 1.9.0mt10, same as is recommended to be used to build Minetest 5.7.

BTW, for getting the touchscreen UI on "desktop" Linux which I assume you'd want on a pinephone, you'd wanna build ROllertest with `ENABLE_TOUCH=TRUE` passed to cmake, in case you're wondering.
